3.3/examples/django/demo.md

```diff
@@ -0,0 +1,73 @@
+# Demo cheat sheet
+
+## Start OpenShift in the background
+
+	sudo /data/src/github.com/openshift/origin/_output/local/go/bin/openshift start --public-master=localhost
+
+Wait until OpenShift is ready.
+
+Open browser at https://localhost:8443.
+
+
+## Allow OpenShift client to reach the OpenShift master
+
+	export OPENSHIFTCONFIG=openshift.local.config/master/admin.kubeconfig
+	export CURL_CA_BUNDLE=openshift.local.config/master/ca.crt
+	sudo chmod a+rwX "$OPENSHIFTCONFIG"
+
+
+## Allow demo user to observe the default namespace
+
+	osadm policy add-role-to-user view demo
+
+
+## Deploy a private docker registry
+
+	sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
+	openshift ex registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig
+
+Wait until the registry is ready.
+
+
+## Create demo project/namespace and give admin role to demo user
+
+	osc new-project demo --display-name="OpenShift Django Demo" --description="This is a demo project featuring Python, Django and PostgreSQL"
+	osadm policy add-role-to-user admin demo
+
+
+## Process template and create entities
+
+Normally you would login before creating the project and processing the template.
+
+	osc process -f application-template-stibuild.json | osc create -f -
+
+
+## Wait for build to finish
+
+	osc get builds --watch
+
+
+## Interact with the application
+
+Forward service port to host machine:
+
+	# get IP and port:
+	osc get service frontend -t '{{ with .spec }}{{ .portalIP }}:{{ with index .ports 0 }}{{ .port }}{{ end }}{{ end }}'
+	# forward port to host:
+	vagrant ssh -- -L 8001:172.30.225.14:5432 -N
+
+Run app management commands:
+
+	./run-in-app-container.sh ./manage.py collectstatic
+	./run-in-app-container.sh kill -s SIGHUP 1          # restart gunicorn
+
+	./run-in-app-container.sh ./manage.py migrate
+	./run-in-app-container.sh ./manage.py createsuperuser
+
+
+## Scale
+
+	# edit first frontend replication controller
+	osc edit rc $(osc get rc -l 'name=frontend-rc' -t '{{ with index}}{{ .metadata.name }}{{ end }}')
+	osc edit rc frontend-1     # change currently running replication controller for immediate scaling
+	osc edit dc frontend       # changing deployment config will affect only next deploy
```

Comment:
I agree it was useful during demo, but since we're going to maintain this example, I'd rather have one readme file with everything in it than two file that might diverge in the future.